src/probnum/quad/__init__.py

```diff
@@ -39,13 +40,16 @@
     "KernelEmbedding",
     "GaussianMeasure",
     "LebesgueMeasure",
-    "StoppingCriterion",
+    "BQStoppingCriterion",
+    "IntegralVarianceToleranceStoppingCriterion",
+    "MaxNevalsStoppingCriterion",
+    "RelativeMeanChangeStoppingCriterion",
 ]
 
 # Set correct module paths. Corrects links and module paths in documentation.
 BayesianQuadrature.__module__ = "probnum.quad"
+BQStoppingCriterion.__module__ = "probnum.quad"
```

Comment:
I would not move component interfaces up to `quad` but keep them in `quad.stopping_criteria`